.github/workflows/examples.yml

```diff
@@ -108,3 +108,12 @@ jobs:
         python examples/pytorch/pytorch_checkpoint.py > /dev/null
       env:
         OMP_NUM_THREADS: 1
+    - name: Run PyTorch checkpoint example (ask-and-tell)
+      run: |
+        # Unset errexit temporarily since timeout returns exit code 124.
```

Comment:
Is this to download the FashionMNIST dataset?

Reply:
No, it isn't. It is for `timeout` command.
The `timeout` command returns non-zero exit code when the timeout happens, and the code stops the script execution without `set +e`.